### PR TITLE
DOC: Add the 2.8 changes to the what's new section

### PR DESCRIPTION
**Pull request by aragilar**

### CODE REVIEW

File: docs/whatsnew/2.8.rst [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  master. The h5py 2.8 release now actually includes these changes.
+* Use lazy-loading of run_tests to avoid strong dependency on unittest2
+  (:issue:`1013`, :issue:`1014` by Thomas VINCENT)
+* Correctly handle with multiple float types of the same size (:issue:`926`
+  by James Tocknell)
+
+Acknowledgements and Thanks
+---------------------------
+The h5py developers thank Nathan Goldbaum, Matthew Brett, and Christoph Gohlke
+for building the wheels that appear or PyPI.
```
Comment (tacaswell) [inline docs/whatsnew/2.8.rst]: or -> on

File: docs/whatsnew/2.8.rst [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+---------------------------------
+There were a number of improvements to testing and CI systems of h5py, including
+running the CI against multiple versions of HDF5, improving reliability and
+speed of the CIs, and simplifying the tox file. See :issue:`857`, :issue:`894`,
+:issue:`922`, :issue:`954` and :issue:`962` by Thomas A Caswell and James
+Tocknell for more details.
+
+Other changes
+-------------
+* Emphasise reading rather than in Quickguide (:issue:`609`, :issue:`610` by Yu
```
Comment (tacaswell) [inline docs/whatsnew/2.8.rst]: rather than what?

**Comment by tacaswell** [comment]
Thanks for doing this!

File: docs/whatsnew/2.8.rst [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+API changes
+-----------
+* Deprecation of ``h5t.available_ftypes``.  This is no longer used internally
+  and will be removed in the future.  There is no replacement public API. See
+  :issue:`926` for how to add addition floating point types to h5py.
+* Do not sort fields in compound types (:issue:`970` by James Tocknell).  This
+  is to account for changes in numpy 1.14.
+* Minimum required version of Cython is now 0.23.
+
+Wheels HDF5 Version
```
Comment (tacaswell) [inline docs/whatsnew/2.8.rst]: Move this to the top?

**Comment by aragilar** [comment]
Added the suggestions, though travis seems to be playing up again, I'll try restarting the individual runs...

**Pull request merged**